### PR TITLE
Fixing DST

### PR DESCRIPTION
**Pull request by mmoczulski**
I opened this PR to do review for you @ninsy  :wink:

### CODE REVIEW

File: lib/DayColumn.js [hunk 1]
```diff
@@ -518,7 +518,10 @@ var _initialiseProps = function _initialiseProps() {
       var minOffset = new Date(min).getTimezoneOffset()
       var initialOffset = new Date(initial).getTimezoneOffset()
       var currentOffset = new Date(current).getTimezoneOffset()
-      if (minOffset > currentOffset || initialOffset < minOffset) {
+      var isFromSummerToWinter =
+        minOffset !== currentOffset && new Date(min).getMonth() === 9
+      debugger // eslint-disable-line
```
Comment (mmoczulski) [inline lib/DayColumn.js]: debugger!

File: lib/utils/dates.js [hunk 2]
```diff
@@ -95,6 +99,9 @@ var dates = _extends({}, _dateArithmetic2.default, {
     return days
   },
   merge: function merge(date, time) {
+    // date = moment(date).tz('Europe/Zurich');
```
Comment (mmoczulski) [inline lib/utils/dates.js:74]: looks like we don't need these comments, do we?

**Comment by mmoczulski** [comment]
Overall, it's still not really good. I try to create event from 9 to 12. This is what I see: 9 to 11 (ignore my accidental click in the beginning of gif). Hours on event popup match ones I saw on my selection. 

agenda-dst

File: lib/DayColumn.js (hunk 1 above)
Comment (pawelrychlik) [reply lib/DayColumn.js]: a ha! ;)

**Comment by mmoczulski** [comment]
doesn't seem to be needed anymore ...

**Pull request closed without merging**